header.php

```diff
@@ -84,18 +84,18 @@
 			</div>
 		<?php } ?>
 
-		<?php if ( 'above' === spacious_options( 'spacious_header_image_position', 'above' ) ) {
+		<?php if ( 'above' === get_theme_mod( 'spacious_header_image_position', 'above' ) ) {
 			spacious_render_header_image();
 		} ?>
 
-		<div id="header-text-nav-container" class="<?php echo ( spacious_options( 'spacious_one_line_menu_setting', 0 ) == 1 ) ? 'menu-one-line' : ''; ?>">
+		<div id="header-text-nav-container" class="<?php echo ( get_theme_mod( 'spacious_one_line_menu_setting', 0 ) == 1 ) ? 'menu-one-line' : ''; ?>">
 
 			<div class="inner-wrap" id="<?php echo esc_attr( $header_class ); ?>">
 
 				<div id="header-text-nav-wrap" class="clearfix">
 					<div id="header-left-section">
 						<?php
-						if ( ( 'both' === spacious_options( 'spacious_show_header_logo_text', 'text_only' ) || 'logo_only' === spacious_options( 'spacious_show_header_logo_text', 'text_only' ) ) ) { ?>
+						if ( ( 'both' === get_theme_mod( 'spacious_show_header_logo_text', 'text_only' ) || 'logo_only' === get_theme_mod( 'spacious_show_header_logo_text', 'text_only' ) ) ) { ?>
```

Comment:
No need to use Yoda condition for function. If you're going to use it, be consistent in the above code as well. 
https://knowthecode.io/yoda-conditions-yoda-not-yoda